nebula-query-and-search/main/classes/Query.cls

```diff
@@ -239,6 +247,11 @@ global class Query extends SOQL {
   //return this.setHasChanged();
   //}
 
+  global Query withAccessLevel(System.AccessLevel mode) {
+    super.doWithAccessLevel(mode);
```

Comment:
Same thing here - for consistency, I think the variable should be called `accessLevel` instead of `mode`

```suggestion
  global Query withAccessLevel(System.AccessLevel accessLevel) {
    super.doWithAccessLevel(accessLevel);
```